lib/connect.js

```diff
@@ -180,6 +180,17 @@ function connect(token, options) {
     ].join(' '));
   }
 
+  if (guessBrowser() === 'edge'
+    && !didPrintSafariOrEdgeWarning
+    && (log.logLevel !== 'error' && log.logLevel !== 'off')) {
+    didPrintSafariOrEdgeWarning = true;
+    log.warn([
+      'This release of twilio-video.js includes experimental support for',
+      'Edge build 10547 and newer. Please test this release',
+      'and report any issues to https://github.com/twilio/twilio-video.js'
```

Comment:
How did you determine the build number? Also, I think it would be good if @ktoraskartwilio weighed in here. IMO, in terms of support, Chrome and Firefox are "tier 1", Safari is "tier 2", and Edge may be "tier 3" or below. I'd like this warning to reflect that, at least at the time of writing, Edge is perhaps "more experimental" than Safari ever was.

Reply:
Our implementation is based on Edge's ORTC API's and ORTC API's are supported on Edge since 10547.